lib/main.dart

```diff
@@ -33,13 +37,19 @@ Future<Config> loadConfig(HttpService http) async {
 
 Future<ServiceRepository> bootstrap() async {
   const http = HttpService();
+  const secureStorage = SecureStorageService(FlutterSecureStorage(
+    aOptions: AndroidOptions(
+      encryptedSharedPreferences: true,
+    ),
+  ));
```

Comment:
this formatting is so weird, isn't it? but apparently dart formatter says it's correct

Reply:
updated